misc/http_client.nim

```diff
@@ -0,0 +1,30 @@
+# nim-raft
```

Comment:
How is this file related to the project? It look like some personal code experiment.

Reply:
Right I will remove it.